src/apps/eth/eth.h

```diff
@@ -31,13 +32,17 @@
  * @param[out] out will hold the result.
  * @param[in] out_len must be at least APP_ETH_ADDRESS_HEX_LEN for ADDRESS, and at least 80
  * (BIP32_SERIALIZED_LEN+1) for XPUB.
+ * @param[in] display Wether to ask user to confirm address
+ * @param[in] contract_address pointer to contract address
```

Comment:
"Must be 20 bytes. If none present, must be 20 zero bytes".

or have the commander do the 20 zero bytes check and pass in NULL in that case (would fit there as it's protobuf input parsing related).